src/store/index.js

```diff
@@ -0,0 +1,81 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+import * as utils from './utils';
+
+Vue.use(Vuex);
+
+
+export default new Vuex.Store({
+  state: {
+    firstStart: true,
+    APIkey: {
+      CLIENT_ID: 'zovm6afV1sSFyuLDNutQn',
+      CLIENT_SECRET: 'Zeee2pQAi37pofB9i6oYRq1b0iepGFc9r4cAjk43',
+    },
+    currentPosition: {},
```

Comment:
Why not
```js
currentPosition: null,
// or
currentPosition: undefined,
// or
currentPosition: { 
  latitude: null, // this allows to see exact data structure
  longitude: null,
},
```
Empty object is not good because it's hard to determine is there is something inside and you need to check code below to determine data structure

Reply:
I think, using empty object is better then `null` or `undefined`, because it's clear now what data type of property to expect. I will use exact data structure.